README.md

```diff
@@ -1,3 +1,12 @@
+This plugin allows you to run Pulumi apps as jobs in your Spinnaker instance. 
+
+## Version Compatibility
+
+| Plugin  | Spinnaker Platform |
+|:----------- | :--------- |
+| 0.1.x  |  1.19.x - 1.22.x |
+| 0.2.x  | 1.23.x |
```

Comment:
I have added a version compatibility table to the README now. I would like to bump the minor version again since the changes in this PR are only for 1.23.x (and above) of Spinnaker.

Reply:
Makes sense